### PR TITLE
[12.0][MIG] account_invoice_tier_validation

### PR DESCRIPTION
**Pull request by patrickrwilson**
Migrating from version 11 to 12.

### CODE REVIEW

**Comment by patrickrwilson** [comment]
ping @osi-scampbell @max3903

**Comment by bodedra** [comment]
@patrickrwilson Please respect commit history. https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

**Comment by patrickrwilson** [comment]
> Please respect commit history. https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

Thanks @bodedra, this is my first migrated module so this will help. This PR can probably be considered 'Work in Progress' and tagged as such. I'll review the documentation, is there a particular area in this module that I should focus on where I didn't meet the requirements?

Also, I know I'll have to fix travis errors too.

**Comment by bodedra** [comment]
Thanks @patrickrwilson You are doing a awesome contribution. Tags/ Labels can be assign by repository maintainers. But we can create a draft PR until travis green.

For a module migration, you can refer https://github.com/OCA/maintainer-tools/wiki#migration

For a module standard, you can setup https://github.com/OCA/maintainer-tools

File: account_invoice_tier_validation/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/11.0/account_invoice_tier_validation
```
Comment (nikul-serpentcs) [inline account_invoice_tier_validation/README.rst]:
```suggestion
    :target: https://github.com/OCA/account-invoicing/tree/12.0/account_invoice_tier_validation
```

File: account_invoice_tier_validation/README.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/account-invoicing <https://github.com/OCA/account-invoicing/tree/11.0/account_invoice_tier_validation>`_ project on GitHub.
```
Comment (nikul-serpentcs) [inline account_invoice_tier_validation/README.rst]:
``` suggestion
This module is part of the `OCA/account-invoicing <https://github.com/OCA/account-invoicing/tree/12.0/account_invoice_tier_validation>`_ project on GitHub.
```

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: account_invoice_tier_validation/README.rst (hunk 1 above)
Comment (patrickrwilson) [reply account_invoice_tier_validation/README.rst]: Fixed, thanks

File: account_invoice_tier_validation/README.rst (hunk 2 above)
Comment (patrickrwilson) [reply account_invoice_tier_validation/README.rst]: Fixed, thanks

**Comment by jonerikceberio** [comment]
It needs to add an exception on access_token, because the dependency on portal.mixin will try to write that field on the account.invoice before the process is finished, therefore making it impossible to validate.
See https://github.com/OCA/sale-workflow/pull/1386/commits/83b864dafb3c213796a3db6570e78923ece79860

**Comment by astirpe** [comment]
@patrickrwilson are you going to continue with this PR?
BTW the fix regarding the `access_token` was included in `base_tier_validation`, so no need to include it in this PR also.

**Comment by patrickrwilson** [comment]
@astirpe I would like to finish this, I just need to find time to figure out how to include commit history from previous version and haven't been able to dive into that yet. This is my first migration and I'm not very familiar with Git and that functionality.

**Comment by astirpe** [comment]
@patrickrwilson I also see tests failing.

**Comment by astirpe** [comment]
@patrickrwilson do you mind if I open another PR to integrate the migration you made with the missing commit history?

**Comment by patrickrwilson** [comment]
@astirpe not at all, I just haven't had the time to work on this so feel free to help out to move this forward. Thanks.

**Comment by astirpe** [comment]
Thanks! I opened https://github.com/OCA/account-invoicing/pull/871 that supersedes this one.

**Comment by AaronHForgeFlow** [comment]
Closing as superseded.

**Pull request closed without merging**